lib/bundler/cli/doctor.rb

```diff
@@ -61,6 +62,7 @@ def check!
     end
 
     def run
+      check_home_permissions
```

Comment:
I'm not sure if this is the best place to put the new home permissions check, it just jumped out to me as the most convenient. I'm happy to move it around if there is a more appropriate place.